#goal definition
geometry_msgs/PoseStamped goal_pose
---
#result definition
geometry_msgs/PoseStamped achieved_pose
---
#feedback
geometry_msgs/PoseStamped current_pose